### ISSUE TITLE
$limit parameter in MostLiked and MostViewed

### ISSUE THREAD

**Issue by jrbecart** [fraank/ViewCounter#3]
Hello,
How to use the $limit parameter in scopeMostLiked and scopeMostViewed function?
I don't know what is it for.

**Comment by fraank** [comment]
You're right, the parameters are not used yet. I wanted to use them for limiting results when the model returns a huge dataset. I will implement it soon. Thanks for reviewing it!
